README.md

```diff
@@ -61,6 +61,29 @@ The general use case for the tool takes following steps:
           > Check what is the latest version in [releases](https://github.com/abstracta/wiresham/releases)
           
           > Check [VirtualTcpServiceTest](src/test/java/us/abstracta/wiresham/VirtualTcpServiceTest.java) and [VirtualTcpClientTest](src/test/java/us/abstracta/wiresham/VirtualTcpClientTest.java) for simple and raw examples on how to use the classes.
+
+## Must-know features
+
+### Multiple port
+There are some scenarios where we need to mock several services under the same domain but differing from port.
+For such scenarios multiple port support was added.
+Here there is a flow example on how a YAML would look:
+
+```yaml
+- !server {data: FFFF, delayMillis: 10, port: 2324}
+- !client {data: FFFF}
+- !server {data: FFFF, port: 2325}
+- !client {data: FFFF}
+- !server {data: FFFF}
+```
+
+**Important considerations:**
+
+1. Connections can be established at any time
+1. Dump is read sequentially. Meaning that any packet received out of order will be ignored (parallelism not yet supported)
+1. When a port is defined, subsequent packets until another port is defined will use the mentioned port without having to explicitly define it (as shown in the example)
+1. Wireshark dumps are parsed using multiple port
```

Comment:
Are there any special considerations for wireshark dumps? When you specify server address it grabs all packets for that server in any port? or how does it "filter" which packets it is going to mock (since usually wireshark dumps include traffic from multiple endpoints)?

Additionally you may want to include tcpdump as well in this statement